### PR TITLE
viz index & a11y section

### PR DESCRIPTION
**Pull request by nhsbsa-Will**

### CODE REVIEW

File: src/data-visualisation/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: article
```
Comment (WORTI3) [inline src/data-visualisation/index.md:2]: Should be marked as `collection`.. even if its draft and not being shown.. we should use the intended layout.

File: src/data-visualisation/accessibility/design-a11y.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+---
```
Comment (WORTI3) [inline src/data-visualisation/accessibility/design-a11y.md:1]: If you are going to have directories within the `data-visualisation` folder you will need to add an `index.md` to each sub-directory. This way when clicking into the "Data visualisation standards" section you'd see a screen filled with your articles.

File: src/data-visualisation/accessibility/design-a11y.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- For amounts in millions or billions, format as '£X million' or '£X billion', not written in full 
+- Do not use '£0.XX millions' for amounts less than £1 million  
+
+::: details Source
+ 
+[Content Design Team ][numbers 1]
+
+::: 
+
+
```
Comment (WORTI3) [inline src/data-visualisation/accessibility/design-a11y.md:47]: Not a biggy but - markdown format usually follows a single consecutive blank line between text/elements.

File: src/data-visualisation/accessibility/design-a11y.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  
+It is important to remember that you must not use colour as the only way to communicate something. Patterns can be used, but they can be cluttering and harder to see. It is always recommended to have an alternative way to gain the same information, for example, by having figures available to the user.  
+  
+::: details Source
+ 
+[WCAG 2.1 Use of Colour][use of colour]
+
+:::
+
+
```
Comment (WORTI3) [inline src/data-visualisation/accessibility/design-a11y.md:64]: Same as above - check all files for double spaces

File: src/data-visualisation/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+title: "Data visualisation standards"
+description: "Data visualisation standards at the NHSBSA"
+status: DRAFT
+tags: [home, data-viz-home]
+order:
+  home: 60
+  data-viz-home: 1
+related:
+  tag: data-viz-home
```
Comment (WORTI3) [inline src/data-visualisation/index.md:11]: If the tag `data-viz-home` is defined in the tags part is it needed here?

File: src/data-visualisation/accessibility/design-a11y.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    
+[Back to Accessibility home](../a11y/)
+
+
+
+[home office posters]: https://github.com/UKHomeOffice/posters/blob/master/accessibility/dos-donts/posters_en-UK/accessibility-posters-set.pdf
+[gov 1]: https://analysisfunction.civilservice.gov.uk/policy-store/making-spreadsheets-accessible-a-brief-checklist-of-the-basics/
+[gov 2]: https://analysisfunction.civilservice.gov.uk/policy-store/releasing-statistics-in-spreadsheets/
+[microsoft]: https://support.microsoft.com/en-us/office/make-your-excel-documents-accessible-to-people-with-disabilities-6cc05fc5-1314-48b5-8eb3-683e49b3e593
+[numbers 1]: https://nhsbsauk.sharepoint.com/sites/DigitalContentDesignTeam/SitePages/NHSBSA-digital-style-guide-and-standards.aspx
```
Comment (WORTI3) [inline src/data-visualisation/accessibility/design-a11y.md:104]: Link seems to be pointing to an internal site. Don't think we want to include these @yantantether ?

File: src/data-visualisation/accessibility/a11y.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+status: DRAFT
+tags: [data-viz, data-viz-a11y, data-viz-home]
+order:
+    data-viz: 2
+    data-viz-a11y: 1
+    data-viz-home: 3
+related: 
+    tag: data-viz-a11y
+---
+## Why do we need to make our products accessible?
```
Comment (WORTI3) [inline src/data-visualisation/accessibility/a11y.md:14]: Check markdown format of entire file

File: src/data-visualisation/accessibility/a11y.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+title: "Data visualisation accessibility"
+description: "Accessibility standards for data visualisation at the NHSBSA"
+status: DRAFT
+tags: [data-viz, data-viz-a11y, data-viz-home]
+order:
+    data-viz: 2
+    data-viz-a11y: 1
+    data-viz-home: 3
+related: 
+    tag: data-viz-a11y
```
Comment (WORTI3) [inline src/data-visualisation/accessibility/a11y.md:12]: Is `data-viz-a11y` needed here when its defined in "tags"?

File: src/data-visualisation/accessibility/a11y.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+---
+## Why do we need to make our products accessible?  
+
+Legally as part of the public sector we are required by law to meet accessibility requirements for both internal and external users.  
+  
+
+We also want to ensure all our users can access and use our products easily and without hinderance. This means that if an individual has an illness, impairment, or a disability, they will be able to use our products. We should prioritise making something accessible without alternative options, with an alternative option being a second resort, to ensure the user can obtain the same information. For example, you would not make text small assuming someone could use a magnifier.
```
Comment (WORTI3) [inline src/data-visualisation/accessibility/a11y.md:19]: hindrance

File: src/data-visualisation/accessibility/design-a11y.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+* Remove gridlines 
+* Minimum colours
+* Don’t have a busy chart  
+- Include alt text (for people with low vision or those who find charts confusing)
+- Include the source of your data, to allow users to familiarise themselves with the data and aid data literacy
+- No wonky text, horizontal is best
+- Don’t use italics, this can be hard for a user to read
+- Make sure your product adapts for smaller screens. This is another reason why a text alternative for a chart is recommended
+- Sans serif font is more understandable for people with dyslexia. Make sure your chart text is as large as is reasonable
+- Don’t mislead the user, for example, start a chart’s y axis from 0 can help ensure the user understands the data clearly
```
Comment (WORTI3) [inline src/data-visualisation/accessibility/design-a11y.md:81]: .."starting a chart's y-axis from 0"...

**Review by WORTI3 — CHANGES_REQUESTED** [review]
Overall content looks good and pages follow playbook structure. 
Please see comments 😄

File: src/data-visualisation/accessibility/a11y.md (hunk 7 above)
Comment (isobelthompson) [reply src/data-visualisation/accessibility/a11y.md:8]: Not sure what the issue is here

File: src/data-visualisation/accessibility/design-a11y.md (hunk 6 above)
Comment (isobelthompson) [reply src/data-visualisation/accessibility/design-a11y.md:93]: I have added a caveat where this link appears on the page.

File: src/data-visualisation/accessibility/design-a11y.md (hunk 2 above)
Comment (isobelthompson) [reply src/data-visualisation/accessibility/design-a11y.md:1]: Implemented this

**Review by WORTI3 — APPROVED** [review]
Looks fine to me. 
Approved on the basis that it is all subject to change as it looks to be getting merged to a feature branch.
@yantantether or @nhsbsa-Will  please could you double check this before merging.

**Pull request merged**